Sources/LeafKit/ParameterResolver.swift

```diff
@@ -115,7 +115,7 @@ struct ParameterResolver {
     private func resolve(op: Operator, rhs: LeafData) throws -> LeafData {
         switch op {
         case .not:
-            let result = rhs.bool ?? false
+            let result = rhs.bool ?? !rhs.isNull
```

Comment:
Would this make sense as an extension on `LeafData` so we don't need to repeat it?

```swift
extension LeafData {
    var isTruthy: Bool {
        self.bool ?? !self.isNull
    }
}
```

Reply:
We might also consider supporting `1` or `"true"` as truthy.